### DISCUSSION TITLE
Metrics route authentication

### DISCUSSION THREAD

**Discussion by Hazmi35** [cloudamqp/lavinmq#606] [General]
Is there a way to authenticate into /metrics? Well you could, set a cookie header, but is there anything like Basic config or something?

**Comment by viktorerlingsson** [comment]
Hi @Hazmi35 

Yes, you can authenticate into /metrics by providing a Basic authentication header. Base64 encode a string of your user:password and pass it along as `Authorization: Basic <encoded string>`. 


https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#basic_authentication
